rpc/systema/agent2systema.proto

```diff
@@ -2,19 +2,22 @@ syntax = "proto3";
 
 package telepresence.systema;
 
+import "google/protobuf/empty.proto";
 import "rpc/manager/manager.proto";
 
 option go_package = "github.com/telepresenceio/telepresence/rpc/v2/systema";
 
-// How Calls from the agent to System A are authenticated is TBD.
+// How Calls from the agent to System A are authenticated is via apikey.
```

Comment:
This sentence is now awkwardly worded.  The sentence was there for symmetry with the multiple comments in `manager2systema.proto`:
```
// Calls from the X to Y are authenticated via RFC 6750
// Bearer tokens.
```
But as I paste that, I realize that comment is outdated; it isn't using RFC 6750 Bearer tokens any more.  So probably update those comments too.

So maybe "Calls from the X to Y are authenticated via API keys per https://www.notion.so/datawire/Telepresence-System-A-API-Key-Integration-c13c4676e9944abfb02713b8517051e3 (see also: `pkg/client/connector/auth/apikey.go`, `cmd/traffic/cmd/manager/systema.go:systemaCredentials`)."? I know it's a faux pas to link to non-public things, but... no one outside of Ambassador Labs is going to be hacking on auth with Ambassador Cloud, and it's the best reference for us when we do hack on it.